todo-sync/data_sources/mongodb-atlas/config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "mongodb-atlas",
+    "type": "mongodb-atlas",
+    "config": {
+        "clusterName": "Cluster0",
```

Comment:
Just thought about this - will Realm handle cluster names on the server side when it creates the template app instance? What if they use a name other than Cluster0?

Reply:
@andrewzlchen any thoughts?